cmd/benchmark/main.go

```diff
@@ -123,10 +127,13 @@ func startScenario(ctx context.Context, s scenario) {
 	writeToOutput(s.getOut(), fmt.Sprintf("Scenario %s: Start to transfer\n", s.getName()))
 
 	start := time.Now()
+	logReaderCloseChan := nodesErrorLogReader(s)
```

Comment:
remind me why do we need `tail -f` mode? instead of just reading logs after benchmark?

Reply:
benchmark runs in loop and generate a lot of logs, so 
1) it's double/triple/etc work 
2) I suppose you want to see errors in the same run of benchmark, that produce it

Reply:
also, it is possible to have errors on network start, we don't wanna see it